### PR TITLE
Add examples to plugin docs

### PR DESCRIPTION
**Pull request by AfiMaameDufie**
Add Examples to Plugin Docs 

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] I added examples of the input plugin with parameters.

### CODE REVIEW

**Comment by AfiMaameDufie** [comment]
@MarkEWaite Can you kindly take a look at this?

File: README.md [hunk 1]
```diff
@@ -1,6 +1,39 @@
...
+
+   input id: '2', message: 'input-message'
+
+
+Input Example with `OK Button Caption`(Optional):
+
+   input message: 'input-message', ok: 'OK'
+
+Input Example with `Allowed Submitter`:
+User IDs and/or external group names of person or people permitted to respond to the input, separated by ','. Spaces will be trimmed. This means that "alice, bob, blah " is the same as "alice,bob,blah".
```
Comment (kmartens27) [inline README.md:26]:
```suggestion
User IDs and/or external group names of a person or people permitted to respond to the input, separated by ','. Spaces will be trimmed automatically, so "Alice, Bob, Charles" is the same as "Alice,Bob,Charles".
```
Changed the sentence structure to be a bit more clear on what will happen, added in a few words to make things read smoothly

**Review by kmartens27 — COMMENTED** [review]
everything looks great! updated the text to be more inline with writing standards, but all the content is wonderful!

**Comment by AfiMaameDufie** [comment]
@kmartens27 Hi Kevin, can you kindly review the change?

**Comment by kmartens27** [comment]
Just reviewed @AfiMaameDufie everything looks awesome, thanks so much!

**Review by rsandell — APPROVED** [review]

**Pull request merged**